docs/docs/deploying-to-firebase.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Deploying to Firebase Hosting
+---
+
+In this guide, you will learn how to deploy your Gatsby site to Firebase Hosting.
+
+[Firebase Hosting](https://firebase.google.com/docs/hosting) is a free web content hosting for developers. With a single command, you can quickly deploy web apps and serve both static and dynamic content to a global CDN (content delivery network).
+
+## Pre-requisites
+
+1. Make sure you have:
+
+   - installed [Node.js](https://nodejs.org/en/download/) on your local machine.
+   - a [Firebase Account](https://console.firebase.google.com).
+
+1. This guide assumes that you already have a Gatsby project on your local machine.
+
+## Deployment
+
+1. Install the Firebase CLI with `npm` by running the following command:
+
+   ```shell
+   npm install -g firebase-tools
+   ```
+
+1. Sign into Firebase using your Google account by running the following command:
+
+   ```shell
+   firebase login
+   ```
+
+   You can test if the CLI is correctly installed by running `firebase projects:list`, which should show you a list of your Firebase Projects.
+
+1. Navigate into your gatsby project directory and setup firebase:
+
+   ```shell
+   firebase init
+   ```
+
+   This command will prompt you to:
+
+   - select the Firebase products you wish to setup. Be sure to select **Firebase Hosting**.
+   - select the Firebase Project you wish to use or create a new one, if you haven't done it previously.
+
+   When prompted to select your public directory, you can simply press <kbd>enter</kbd> since it defaults to `public`, which is also Gatsby's default public directory.
```

Comment:
```suggestion
   When prompted to select your public directory, press <kbd>enter</kbd>. It will default to `public`, which is also Gatsby's default public directory.
```